ls1intum/Artemis
# `Assessment`: Escape the participant name in the csv format of the scores export to avoid issues with commas

### Checklist
#### General


#### Client

### Motivation and Context
Closes #4767

### Description
Quotes are added around the student/team name.
The other fields (login name, score, points, repository-URL) never contain commas. Therefore, putting quotes around them is not necessary.

### Steps for Testing

Prerequisites:
- 1 Instructor
- Some exercise with student submissions

1. Go to the Scores page of the exercise.
2. Click on the Export Scores button in the top-right corner.
3. The name of the student/the team name should be inside `"`s. You might have to open the file in a text editor instead of LibreOffice/Excel as they probably remove the quotes automatically.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage
unchanged

### Screenshots
n/a
